tests/topotests/lib/common_config.py

```diff
@@ -3497,7 +3503,7 @@ def verify_prefix_lists(tgen, input_dict):
     return True
 
 
-@retry(attempts=3, wait=4, return_is_str=True)
+@retry(attempts=2, wait=4, return_is_str=True, initial_wait=2)
```

Comment:
Why this change and what does it have to do with pim?

Reply:
I have aligned with base changes